src/assets/toolkit/styles/components/sky.css

```diff
@@ -17,6 +16,7 @@
   --Sky-nav-logo-height: 5em;
   --Sky-nav-logo-width: var(--Sky-nav-logo-height);
   --Sky-nav-logo-padding: var(--Sky-nav-controls-space-y) var(--Sky-nav-controls-space-x);
+  --Sky-nav-color-hover: color(var(--Sky-background-color) s(+10%) l(90%));
```

Comment:
Nit: I'm pretty sure this color is applied for everything _but_ hover. Maybe `inactive` would be more descriptive?